ivy_tests/test_ivy/helpers/test_parameter_flags.py

```diff
@@ -232,10 +232,12 @@ def __init__(
         num_positional_args,
         as_variable,
         native_arrays,
+        generate_frontend_arrays,
```

Comment:
We shouldn't add this flag here, instead we should have a separate class for Frontends since the flags are now different.
Having this here would be redundant for method testing for backend.